packages/nimble-components/src/tree-view/tests/tree-view.stories.ts

```diff
@@ -6,6 +6,7 @@ import { treeViewTag } from '..';
 import { iconCogTag } from '../../icons/cog';
 import { iconDatabaseTag } from '../../icons/database';
 import { treeItemTag } from '../../tree-item';
+import { anchorTreeItemTag } from '../../anchor-tree-item';
```

Comment:
We should borrow some patterns from how we documented the anchor menu in the menu story:
- in the description of the tree we should describe the element types that can be children of the tree
- we should have a dedicated story for the anchor tree item describing its API

Reply:
Done